modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManagerToolbar.vue

```diff
@@ -74,7 +76,8 @@ export default {
         field: {
           name: 'search',
           placeholder: 'Search Images',
-          icon: 'magnify-icon'
+          icon: 'magnify-icon',
+          enterSubmittable: true
```

Comment:
The string already had a return event for AposTagApply that didn't return the value. There wouldn't be any harm in always including the value formatted like this (it's the same format as the `input` emission), but this make it an option.